front/pages/w/[wId]/join.tsx

```diff
@@ -0,0 +1,86 @@
+import { Logo } from "@dust-tt/sparkle";
+import { GetServerSideProps, InferGetServerSidePropsType } from "next";
+import { signIn } from "next-auth/react";
+
+import { GoogleSignInButton } from "@app/components/Button";
+import { isWorkspaceAllowedOnDomain } from "@app/lib/api/workspace";
+
+const { URL = "", GA_TRACKING_ID = "" } = process.env;
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
we're missing a bit of typing of the props?

Reply:
Ooops indeed! I added a commit!